packages/webpack/lib/configs/build.js

```diff
@@ -141,8 +141,9 @@ module.exports = function getConfig(config, name) {
       new EnvironmentPlugin({ NODE_ENV: 'development' }),
     ],
     performance: {
```

Comment:
Should @untool provide such "opinionated" defaults?
And also, is there a reason why we configure `performance` field and we keep it turned off with `hints: false`?

Reply:
I think it should - in fact, it always did: Webpack sets both values to 250000, I simply bumped it to 262144 (= 256kB = 0,25MB). I disabled the performance warning output to be able to handle it in `untool` directly: 

https://github.com/untool/untool/blob/007f3e160f5e0b132aec843de6e45dab99c48912/packages/yargs/lib/webpack.js#L17-L28